assets/espresso/5.2.1.yml

```diff
@@ -0,0 +1,27 @@
+#  structure:
```

Comment:
1. The declaration of the versions is in https://github.com/Exabyte-io/application-flavors/blob/8ee57eb9468a492da14bf35782544a342f0fd4dd/src/js/espresso/data.js, should we move assets there?
2. We should consider how the connection between path and flavor works

Reply:
1. Moved code to application-flavors: https://github.com/Exabyte-io/application-flavors/pull/63
2. Created task for this purpose.